install.sh

```diff
@@ -226,6 +243,7 @@ nvm_do_install() {
     fi
   fi
 
+  nvm_bash_completion
```

Comment:
The logic is definitely flawed and needs improvement - see https://github.com/creationix/nvm/labels/installing%3A%20profile%20detection - but we should be consistent :-)